### DISCUSSION TITLE
Error After Manual Update 4.7 to 4.8

### DISCUSSION THREAD

**Discussion by UkinnovationTechs** [Blair2004/NexoPOS#991] [Q&A] [answered]
Get Error After Manual Update 4.7 to 4.8

modules
 / 
NsMultiStore
 / 
Listeners
 / 
MultiStoreOrderEventsSubscriber
.php
 
: 28

use App\Services\CustomerService;

use App\Services\ExpenseService;

use App\Services\OrdersService;

use App\Services\ProcurementService;

use App\Services\ProductService;

use App\Services\ProviderService;

use Illuminate\Support\Facades\Event;

use Illuminate\Support\ServiceProvider;

use Modules\NsMultiStore\Jobs\StoreComputeCashierSalesJob;

use Modules\NsMultiStore\Jobs\StoreComputeCustomerAccountJob;

use Modules\NsMultiStore\Jobs\StoreComputeDayReportJob;

use Modules\NsMultiStore\Listeners\StoreOrderEventsSubscriber;

use Modules\NsMultiStore\Models\Store;

use Modules\NsMultiStore\Services\StoresService;

class MultiStoreOrderEventsSubscriber extends OrderEventsSubscriber

{

    /**

     * @var StoreOrderEventsSubscriber

     */

    protected $orderEvents;

    /**

     * @var ProductService

     */

    protected $productService;

    /**

     * @var OrdersService

     */

**Comment by Blair2004** [comment, marked as answer]
Hi, bugs should be reported on issue section following the default and understandable format.
